helix-view/src/editor.rs

```diff
@@ -595,6 +599,8 @@ impl Default for Config {
                 GutterType::Spacer,
                 GutterType::LineNumbers,
             ],
+            line_numbers_width_min: 2,
+            line_numbers_width_max: 0, // unlimited
```

Comment:
Currently, setting the maximum to `0` is equivalent to "unlimited". I'm mixed about this decision. I don't particularly like the idea of hard-coding a cap, but I also don't like using `0` as a "magic number". Open to suggestions.